### PR TITLE
Add version query param to createFromTemplate()

### PR DESCRIPTION
**Pull request by deepakrkris**
- [ ] add loopback version as argument to createFromTemplate() remote method

Connect to https://github.com/strongloop/loopback-workspace/issues/448

### CODE REVIEW

File: test/end-to-end.js [hunk 1]
```diff
@@ -32,6 +32,22 @@ var MYSQL_PASSWORD = 'hbx42rec';
 describe('end-to-end', function() {
   this.timeout(15000);
 
+  describe('can pass LoopBack version to createFromTemplate()', function() {
+    before(resetWorkspace);
+    before(givenEmptySandbox);
```
Comment (superkhau) [inline test/end-to-end.js]: nit empty line between before + it

File: test/end-to-end.js [hunk 2]
```diff
@@ -32,6 +32,22 @@ var MYSQL_PASSWORD = 'hbx42rec';
...
+    it('create template 3.x', function(done) {
```
Comment (superkhau) [inline test/end-to-end.js]: it('creates a template based on the given LoopBack version', () => {

File: test/end-to-end.js [hunk 3]
```diff
@@ -32,6 +32,22 @@ var MYSQL_PASSWORD = 'hbx42rec';
...
+      givenWorkspaceFromTemplate('hello-world', '3.x',
+      function(err) {
+        if (err) return done(err);
+        var dependencies = readPackageJsonSync().dependencies;
```
Comment (superkhau) [inline test/end-to-end.js]: const deps = require('../path/to/json').dependencies;

**Comment by anthonyettinger** [comment]
lgtm

File: test/end-to-end.js [hunk 4]
```diff
@@ -32,6 +32,22 @@ var MYSQL_PASSWORD = 'hbx42rec';
...
+        expect(semver.gtr('3.0.0', dependencies.loopback)).to.be.false;
```
Comment (superkhau) [inline test/end-to-end.js]:
```
const lbVersion = deps.loopback;
const isLoopbackVersion3 = semver.satisfies(lbVersion, '^3.0.0');
expect(isLoopBackVersion3).to.be.true();
// ditto for explorer
```

**Pull request merged**